packages/devextreme-react/src/core/options-manager.ts

```diff
@@ -103,7 +114,8 @@ class OptionsManager {
 
     changedOptions.forEach(([name, value]) => {
       const currentPropValue = config.options[name];
-      if (config.options.hasOwnProperty(name) && currentPropValue !== value) {
+      if (Object.prototype.hasOwnProperty.call(config.options, name)
```

Comment:
Could you explain why what this change made for?

Reply:
https://eslint.org/docs/latest/rules/no-prototype-builtins